src/components/policy/policy_external/src/policy_table/enums.cc

```diff
@@ -466,8 +467,6 @@ const char* EnumToJsonString(AppHMIType val) {
       return "TESTING";
     case AHT_SYSTEM:
       return "SYSTEM";
-    case AHT_PROJECTION:
```

Comment:
@yang1070 this is not removed, just order changed. Because order of enums impacts logic.
@dev-gh could you please mention PR with this change and description of fix.